### PR TITLE
Creating recommendations from Houston for Phoenix

### LINKED ISSUE #14: Create a new file for houstongilbert's hometown (Phoenix, AZ)
## Phoenix, AZ
- [x] Create a branch
- [x] Add commits
- [x] Open a pull request 
- [x] Discuss and review (collaboration)
- [x] Seek approval
- [x] Merge PR

### PR DESCRIPTION
**Pull request by houstongilbert**
This represents intent that has not yet been accepted or committed

closes #14

### CODE REVIEW

File: places/houstongilbert-phoenix.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
 # Phoenix, AZ
 
 ### Places to eat
+- La Piñata (North Central)
```
Comment (jerrycheen) [inline places/houstongilbert-phoenix.md:4]: Is it Mexican food or Spanish food

File: places/houstongilbert-phoenix.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Phoenix, AZ
+
+### Places to eat
+- La Piñata (North Central)
```
Comment (jerrycheen) [inline places/houstongilbert-phoenix.md:4]: Sounds a good place to go

Comment (houstongilbert) [reply places/houstongilbert-phoenix.md:4]: It is. Try the house margarita!

File: places/houstongilbert-phoenix.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Phoenix, AZ
```
Comment (pourmoht) [inline places/houstongilbert-phoenix.md:1]: MAKE ARIZONA GREAT AGAIN!!!

Comment (houstongilbert) [reply places/houstongilbert-phoenix.md:1]: Phoenix is the worst

**Review by jerrycheen — APPROVED** [review]

**Pull request merged**